README.md

```diff
@@ -28,6 +28,13 @@ Our [`python-plugwise`](https://github.com/plugwise/python-plugwise) python modu
 
 # Changelog
 
+## NEW MAR 2022 [0.21.4]
+- Smile:
+  - Link to plugwise v0.16.8 - https://github.com/plugwise/python-plugwise/releases/tag/v0.16.8
+  - Bugfixes via updating plugwise: 
```

Comment:
Suggest to include minimal recap of the issues by adding
 - ... when discovery IP resolving to IPv6
 - ... w.r.t. outdoor temperature with auxiliary device

Reply:
Implement, the 2nd bullet text has been made similar to the wording of the previous release.